mxcubeweb/logging_handler.py

```diff
@@ -26,14 +21,10 @@ def _record_to_json(self, record):
             "message": record.getMessage(),
             "severity": record.levelname,
             "timestamp": record.asctime,
-            "logger": record.name,
-            "stack_trace": stack_trace,
```

Comment:
I removed the `logger` and `stack_trace` from the LoggingHandler, as we only need the user_level_log to be shown in the UI and due to some concerns about the stack_trace being sent to the front-end in #1393. Could this removal be confirmed as accepted or is the general opinion to keep it?